src/schedule-and-details/basic-section/index.jsx

```diff
@@ -96,7 +93,11 @@ const BasicSection = ({
 
   const renderCoursePromotion = () => (
     <Card>
-      <Card.Header className="h4 px-3 text-gray-500" title={promotionTitle} size="sm" />
+      <Card.Header
+        className="h4 px-3 text-gray-500"
+        title={promotionTitle}
```

Comment:
Should the `promotionTitle` be wrapped in a translation?

Reply:
we already have it wrapping translate
```
const promotionTitle = (
    <FormattedMessage
      id="course-authoring.schedule.basic.promotion.title"
      defaultMessage="Course summary page {smallText}"
      values={{
        smallText: <small>(for student enrollment and access)</small>,
      }}
    />
  );
```